ci/matrix.yaml

```
@@ -107,7 +107,8 @@ workflows:
     - {jobs: ['test'],  project: 'cudax', ctk: ['12.0'        ], std: 'all', cxx: ['clang14'], gpu: 'rtx2080'}
     - {jobs: ['test'],  project: 'cudax', ctk: [        'curr'], std: 'all', cxx: ['clang'],   gpu: 'rtx2080'}
     # Python and c/parallel jobs:
-    - {jobs: ['test'], project: ['cccl_c_parallel', 'python'], gpu: 'rtx2080'}
+    - {jobs: ['test'], project: ['cccl_c_parallel'], gpu: 'rtx2080'}
```

nit: I haven't figured out how the script is able to tell "a single string" from "a list of one string", I imagine there's a bit of special parsing logic that I haven't discovered. Suggesting this only for consistency:
```suggestion
    - {jobs: ['test'], project: 'cccl_c_parallel', gpu: 'rtx2080'}
```

---

Thanks. I'll tack this onto another PR so as to not kick off the entire test suite again.